fix production build (fixes #941)

Betalos opened the pull request:
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
 The issue was caused with multiple build (dev/prod at the same time)

* **What is the current behavior?** (You can also link to an open issue here)
issue with the log


* **What is the new behavior (if this is a feature change)?**
in the production build the log is removed


* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Update deps (did not cause issues with the build as far as I know)


* **Other information**:

dannyrb commented:
@Betalos, is the solution here to leave logging in `production`, or to conditionally remove `debug` in production builds? I don't follow how this PR achieves the desired outcome

Betalos commented:
@dannyrb the issue was that in package.json build script:
```cmd
npm run test && npm run clean:dist && npm run version && npm run build:prod && npm run build:dev
```
it generate the dev build + prod one (prod with .min extention) this was the issue.
the import from cornerstoreTools would pull from dev build not .min (unless we import directly from it ex: 
```typescript
import * as tools from 'cornerstoneTools/dist/cornerstoneTools.min'
```
so in this PR I removed the dev build and added another entry 'cornerstoneTools.min' and a condition on terser plugin to only minimize the file with .min at the end. With this we achieve the desired outcome, the issue with:
```javascript
var devMode = "production" === 'development'; // If we're in dev-mode, tell the user how to get logs
```
instead of 👍 ```javascript
var devMode = "development" === 'development'; // If we're in dev-mode, tell the user how to get logs
```

Betalos commented:
@dannyrb any progress on this ???
